src/cmd/linuxkit/push_gcp.go

```diff
@@ -52,15 +52,15 @@ func pushGcp(args []string) {
 		name = filepath.Base(name)
 	}
 
-	if bucket == "" {
-		log.Fatalf("Please specify the bucket to use")
-	}
-
 	client, err := NewGCPClient(keys, project)
 	if err != nil {
 		log.Fatalf("Unable to connect to GCP: %v", err)
 	}
 
+	if bucket == "" {
```

Comment:
Why do you move this? could you add this to the commit message?

Reply:
I separated the commits to be more clear 👍 I just want to have a more coherent behaviour between the gcp push and run: the client check the project name and if it fail, output error. And then the specific check for push begins.